modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,153 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with the following procedure.
+
+.Procedure
+
+. Download the {product-title} client (`oc`) and make it available for use by entering the following command:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz > oc.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxf oc.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x oc
+----
+
+. Set the {product-title} version:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version> <1>
+----
++
+<1> Replace `<ocp_version>` with the current version. For example. `latest-{product-version}`
+
+. Download the {product-title} installer and make it available for use:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-install-linux.tar.gz > openshift-install-linux.tar.gz
```

Comment:
Dumb question: @aireilly Will the variable `$OCP_VERSION` resolve correctly inside the URL without any other markup?

Reply:
Yup! works just dandy :)

```
┌───── ~/Downloads 
$ OCP_VERSION=latest-4.9

┌───── ~/Downloads 
$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-install-linux.tar.gz > openshift-install-linux.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 99.4M  100 99.4M    0     0  7932k      0  0:00:12  0:00:12 --:--:-- 10.5M
```